--- a/rel/overlay/bin/couchdb	2017-08-10 08:03:04.701531748 +0200
+++ b/rel/overlay/bin/couchdb	2017-08-10 08:14:31.278729581 +0200
@@ -26,6 +26,11 @@
 export EMU=beam
 export PROGNAME=`echo $0 | sed 's/.*\///'`
 
+[ -z ${COUCHDB_VM_ARGS+x} ] && COUCHDB_VM_ARGS="$ROOTDIR/etc/vm.args"
+[ -z ${COUCHDB_INI_FILES+x} ] && COUCHDB_INI_FILES=
+
 exec "$BINDIR/erlexec" -boot "$ROOTDIR/releases/$APP_VSN/couchdb" \
-     -args_file "$ROOTDIR/etc/vm.args" \
+     -args_file "$COUCHDB_VM_ARGS" \
+     -couch_ini $COUCHDB_INI_FILES \
      -config "$ROOTDIR/releases/$APP_VSN/sys.config" "$@"
+     
